src/MetaDataAPI/Utils/DynamoDb.cs

```diff
@@ -47,22 +37,29 @@ public string PutItem(IEnumerable<Erc721Attribute> attributes)
                     { PrimaryKey, new AttributeValue { S = hash } },
                     { "Data", new AttributeValue { S = jsonAttributes } },
                     { "InsertedTime", new AttributeValue { N = DateTimeOffset.UtcNow.ToUnixTimeSeconds().ToString() } }
-                }
+                },
+                ConditionExpression = $"attribute_not_exists({PrimaryKey})"
             }
         };
 
         var request = new TransactWriteItemsRequest
         {
             TransactItems = new List<TransactWriteItem>
             {
-                new() { ConditionCheck = conditionCheck },
                 new() { Put = put.Put }
             }
         };
 
-        client.TransactWriteItemsAsync(request)
-            .GetAwaiter()
-            .GetResult();
+        try
+        {
+            client.TransactWriteItemsAsync(request)
+                .GetAwaiter()
+                .GetResult();
+        }
+        catch (TransactionCanceledException)
+        {
+            return hash;
```

Comment:
let's make sure the exception is due to the item exists, and not another error
no need to `return hash;` in `catch`, it will do it next line